src/nemos/fetch/fetch_data.py

```diff
@@ -130,48 +85,10 @@ def _create_retriever(path: Optional[pathlib.Path] = None) -> pooch.Pooch:
         registry=REGISTRY_DATA,
         retry_if_failed=2,
         allow_updates="POOCH_ALLOW_UPDATES",
-        env="NEMOS_DATA_DIR",
+        env=_NEMOS_ENV,
```

Comment:
do you have a use case in mind for this?

Reply:
I do use it to have the data in my repo docs/data directory instead of a cache directory harder to locate.